src/contract.rs

```diff
@@ -442,16 +512,23 @@ mod tests {
 
     #[test]
     fn create_transfer_insufficient_funds_throws_error() {
-        let mut deps = mock_dependencies(&[]);
+        let mut deps = mock_provenance_dependencies();
         setup_test_base(
             &mut deps.storage,
             &State {
                 name: "contract_name".into(),
             },
         );
 
-        let test_marker: Marker = setup_restricted_marker();
-        deps.querier.with_markers(vec![test_marker]);
+        let test_marker: MarkerAccount = setup_restricted_marker();
+        let mock_marker_response = QueryMarkerResponse {
```

Comment:
nit: this could be a useful test helper. Something like `mock query marker response`